cmdx.py

```diff
@@ -1,5 +1,16 @@
 # -*- coding: utf-8 -*-
 
+# Hack for static typing analysis:
+# the test below only passes in the IDE (eg VsCode); Maya doesn't know or care about typing.
+MYPY = False
```

Comment:
Can MYPY be called something else, like `_MYPY` or `_TYPINGSUPPORT`? Best not to expose members to `cmdx` unless we intend on others using it and us supporting it. Do we need it, or could it if be `if False:`?

Reply:
No, it has to be MYPY, it's a weird hack where the analyzer looks for this specific token and forcibly considers it true: https://mypy.readthedocs.io/en/stable/common_issues.html#import-cycles
... I agree the naming is unfortunate.

Reply:
Ah, could we at least `del MYMY` either directly after the `if`, or towards the end of the file? To avoid anyone using `cmdx.MYPY`

Reply:
Ah yes we can apparently, done.